packages/cli/test/sim/multi_fork.test.ts

```diff
@@ -153,6 +153,9 @@ const headForUnknownBlockSync = await env.nodes[0].cl.api.beacon.getBlockV2("hea
 ApiError.assert(headForUnknownBlockSync);
 await connectNewNode(unknownBlockSync, env.nodes);
 
+// Wait for EL node to start and sync
+await sleep(5000);
```

Comment:
The `uknownBlock` sync does not trigger to completion if the `EL` node is still syncing. So why had to add some wait time so EL node get sync first then we publish an uknown block.